plugins/bernard.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: default
+permalink: /plugins/bernard/
+title: Bernard
+---
+
+# Bernard
+
+[![Author](http://img.shields.io/badge/author-@sagikazarmark-blue.svg?style=flat-square)](https://twitter.com/sagikazarmark)
+[![Source](http://img.shields.io/badge/source-league/tactician--bernard-blue.svg?style=flat-square)](https://github.com/thephpleague/tactician-bernard)
+[![Packagist](http://img.shields.io/packagist/v/league/tactician-bernard.svg?style=flat-square)](https://packagist.org/packages/league/tactician-bernard)
+
+
+This plugin provides you tools for both sending commands to and consuming from a queue, so you should install it in both your application and (if you have any) dedicated worker package/instance.
+
+You can run your consumer from your application environment, however you should avoid it if possible. Bootstrapping your application consumes more resource than necessary. (That said, sometimes it is simply easier.) In this case you have to make sure that you provide your consumer instance with all the data it needs: database connection details, required dependencies, etc.
```

Comment:
As mentioned elsewhere, I'm not in complete agreement with recommending this to people. These days, "application environment" usually means their DI container and CLI library, which I think they _should_ bootstrap with.